### PR TITLE
Update Getting Started guide to 0.8.0.RC1

### PR DESCRIPTION
**Pull request by jczuchnowski**
Closes #xxx 
___

## Summary:

(please input the summary)

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -4,68 +4,92 @@ sidebar_position: 2
...
 
-1. Add Mesmer Akka extension:
+2. Download [mesmer-otel-extension.jar](https://github.com/ScalaConsultants/mesmer/releases/download/v0.8.0.RC1/mesmer-otel-extension.jar) from `mesmer` Releases.
+
+3. Add Mesmer Akka extension:
 
    Add the following dependency to your `build.sbt` file:
    ```scala
-   libraryDependencies += "io.scalac" %% "mesmer-akka-extension" % "0.7.0"
+   libraryDependencies += "io.scalac" %% "mesmer-akka-extension" % "0.8.1.RC1"
```
Comment (lgajowy) [inline docs/getting-started.md:23]: It's 0.8.0.RC1

File: docs/getting-started.md [hunk 2]
```diff
@@ -4,74 +4,119 @@ sidebar_position: 2
...
 
-- **Akka Extension** - that runs in the background and is responsible for exporting the metrics to your chosen backend
-- **OpenTelemetry Agent Extension** - that instruments Akka classes to expose metrics for the Extension
+This guide presents steps for Prometheus and OTEL (native OpenTelemetry protocol) exporters.
 
-Both parts need to be included in the application for Mesmer to work.
+## Basic setup
 
-## Step by step guide
+1. Download [opentelemetry-javaagent.jar](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v1.12.0/opentelemetry-javaagent.jar) from `opentelemetry-java` Releases.
```
Comment (lgajowy) [inline docs/getting-started.md:15]:
```suggestion
1. Download [opentelemetry-javaagent.jar](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v1.13.1/opentelemetry-javaagent.jar) from `opentelemetry-java` Releases.
```

That's the highest version of opentelemetry we use in our dependencies. I use it on my machine as well

File: docs/getting-started.md [hunk 3]
```diff
@@ -4,74 +4,119 @@ sidebar_position: 2
...
+   
+   **For OTLP**
+
+   You need to have a running OTLP Collector for the metrics to be automatically streamed to it.
+
+   **For Prometheus**
+
+   Call the metrics endpoint:
    ```sh
    curl -i http://localhost:9464
```
Comment (lgajowy) [inline docs/getting-started.md:79]: Does this work? I usually just go to localhost:9090 and query for metrics there.

Comment (jczuchnowski) [reply docs/getting-started.md:79]: but you have Prometheus running, right? localhost:9464 is the endpoint that Prometheus calls to get the metrics from the application

File: docs/getting-started.md (hunk 2 above)
Comment (jczuchnowski) [reply docs/getting-started.md]: :+1:

**Review by lgajowy — APPROVED** [review]

**Pull request merged**